pkgs/by-name/ia/ianny/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, dbus
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ianny";
+  version = "0.1.0alpha.2";
```

Comment:
The latest version is a beta, could you please update?

Reply:
Sure, Done!